### ISSUE TITLE
Movie plays back with yellow/green tint (Digital Glider Airman)

### ISSUE THREAD

**Issue by gingerbeardman** [libretro/beetle-psx-libretro#754]
## Description

Movie plays back with yellow/green tint (Digital Glider Airman)

### Expected behavior

Movie should play back in colour

- ![2020-11-01 00_08_00-Greenshot](https://user-images.githubusercontent.com/49612/97792450-6e280580-1bd6-11eb-97ff-321a5ca19d20.png)

- ![2020-11-01 00_09_24-Greenshot](https://user-images.githubusercontent.com/49612/97792457-89931080-1bd6-11eb-935f-4d860e17419d.png)

### Actual behavior

Movie plays back in green

- ![52a97cda005c1176bfdd6bf91c346826a2f1c41c](https://user-images.githubusercontent.com/49612/97792471-b21b0a80-1bd6-11eb-87b4-3571241cdca9.png)

- ![8150c771e581ed93bb6a06bce27c36e817911040](https://user-images.githubusercontent.com/49612/97792473-b47d6480-1bd6-11eb-8613-7cd125a494e9.png)

### Steps to reproduce the bug

1. load game
2. watch intro
3. compare to elsewhere

Screens above are from End Movie

### Bisect results

[Try to bisect and tell us when this started happening]

### Version/Commit

1.9.0 Beetle PSX HW (0.9.44.1 b432b68)

### Core options

[List all the core options you have changed]

Please also specify, if reporting a graphical glitch for Beetle PSX HW, your "Renderer" core option setting and your frontend selected video driver.

- Renderer: Hardware (Vulkan)

### Environment information

- Frontend: RetroArch
- OS: Win 10
- Compiler: MinGW (9.2.0) 64-bit
- GPU: Intel HD 620

**Comment by hizzlekizzle** [comment]
Same issue? https://github.com/libretro/beetle-psx-libretro/issues/600

**Comment by gingerbeardman** [comment]
@hizzlekizzle yes, thanks. And also from there #467 which implies it's an Intel driver bug?

Regardless, feel free to close in favour of one of those.

**Comment by Jj0YzL5nvJ** [comment]
Maybe this is related? ata4/angrylion-rdp-plus#77

**Comment by gingerbeardman** [comment]
@Jj0YzL5nvJ interesting.

It looks like the Blue channel is almost all missing wrong in the green/yellow tinted output.

# Correct
- <img width="106" alt="Screen shot 2020-11-02 at 17 10 08" src="https://user-images.githubusercontent.com/49612/97897441-624f5700-1d2e-11eb-88ec-c6a87736ff13.png">
- ![Screen shot 2020-11-02 at 17 12 55](https://user-images.githubusercontent.com/49612/97897751-d853be00-1d2e-11eb-80d8-49af67a17d67.png)

# Incorrect
- <img width="112" alt="Screen shot 2020-11-02 at 17 10 18" src="https://user-images.githubusercontent.com/49612/97897448-63808400-1d2e-11eb-8d6a-562ab7afbeca.png">
- ![Screen shot 2020-11-02 at 17 13 03](https://user-images.githubusercontent.com/49612/97897754-d8ec5480-1d2e-11eb-96dc-82a2326182e4.png)

**Comment by hizzlekizzle** [comment]
I don't know that we need to close it in favor of one or the other. I think just having them referenced is sufficient, so if/when we get a fix, we can verify which other issues are closed, too.

**Comment by gingerbeardman** [comment]
Yes, that's why I did not close it.

**Comment by gingerbeardman** [comment]
interestingly the menu screen of Gran Turismo 1, a static screen, also displays with this green/yellow tint.

**Comment by IonBlade2K** [comment]
This has been driving me crazy, and I didn't see anyone looking into it in this or the other thread, so I've gone ahead and thrown $50 on a bounty for it over at https://www.bountysource.com/issues/93781925-movie-plays-back-with-yellow-green-tint-various-games.

**Comment by ZRupp** [comment]
This happens on Android only with the Beetle PSX HW core. If I use the Beetle PSX core with Vulkan it doesn't happen.

**Comment by hizzlekizzle** [comment]
I believe this has been fixed. Can anyone test and verify?

**Comment by Starlightbotanist** [comment]
Fixed on android. Tested with various codemaster games (Colin McRae Rally 2.0, Jarrett & Labonte Stock Car Racing), Gran Turismo, and Motorhead.
